app/main.js

```diff
@@ -246,13 +297,15 @@ function finishMicrobreak (shouldPlaySound = true) {
   if (shouldPlaySound) {
     processWin.webContents.send('playSound', settings.get('audio'))
   }
-  if (microbreakWin) {
+  if (microbreakWins) {
     if (process.platform === 'darwin') {
       // get focus on the last app
       Menu.sendActionToFirstResponder('hide:')
     }
-    microbreakWin.close()
-    microbreakWin = null
+    for (let i = microbreakWins.length - 1; i >= 0; i--) {
```

Comment:
could we create a function out of this?  it would accept array of windows and close them one by one.

Reply:
Yes, I'll create a function `closeDisplays` which takes in a list of displays and closes all of them